### PR TITLE
Pixel Projectiles!!

### PR DESCRIPTION
**Pull request by AliiTea**
@nalar DNM tag this until it's accepted plox

### CODE REVIEW

**Comment by SplergeBucket** [comment]
this that thing where you shoot where you click
thats okay

**Comment by nalar** [comment]
Pretty sure this is an accidental tagging of my (@nalar)

File: tgstation.dme [hunk 1]
```diff
@@ -1487,6 +1487,7 @@
 #include "code\modules\vehicles\lawnmower.dm"
 #include "code\modules\vehicles\segway.dm"
 #include "code\WorkInProgress\atlantique\screambot.dm"
+#include "code\WorkInProgress\crystal\pixelprojectiles.dm"
```
Comment (ghost) [inline tgstation.dme:1490]: "WorkInProgress" it's not WorkInProgress if it's live on the server, stop stuffing shit into the WorkInProgress folder

Comment (Chronitonity) [inline tgstation.dme:1490]: pretty sure its literally a work in progress seeing how vadim never perfected this and wanted to revisit it iirc

**Comment by nalarmg** [comment]
@nalar hello other me. Also yes, it was.

**Comment by AliiTea** [comment]
@chnkr I put it in there for "legacy"

File: tgstation.dme (hunk 1 above)
Comment (nalarmg) [inline tgstation.dme:1490]: It can be live and still be a work in progress.

**Comment by danhibiki** [comment]
I guess this has sat for awhile, but I'll give my approval for a test run. If the server at large doesn't like it, we should consider reverting, but I don't forsee many complaints

**Comment by nalarmg** [comment]
Conflicts. Unable to merge.

**Comment by danhibiki** [comment]
Well, obviously if the conflicts were lifted, I wasn't intending to do anything personally as I do not know how to resolve such things

**Comment by Iamgoofball** [comment]
we test ran this on sybil awhile back

everyone fucking loved it

**Comment by AliiTea** [comment]
@Iamgoofball why didn't you keep it on then?

**Comment by Iamgoofball** [comment]
2lazy

**Comment by Chronitonity** [comment]
@Memendia fix your changelog thats the only conflict i can see

File: html/changelog.html [hunk 2]
```diff
@@ -56,6 +56,14 @@
 <!-- DO NOT REMOVE OR MOVE THIS COMMENT! THIS MUST BE THE LAST NON-EMPTY LINE BEFORE THE LOGJS #ADDTOCHANGELOGMARKER# -->
 
 <div class='commit sansserif'>
+	<h2 class='date'>15 September 2015</h2>
+	<h3 class='author'>Memendia updated:</h3>
+	<ul class='changes bgimages16'>
+		<li class='rscadd'>Ported Crystal's PixelProjectiles!</li>
```
Comment (Iamgoofball) [inline html/changelog.html:62]: > 2015
> using this retarded changelog system still

what are you SMOKING

Comment (ghost) [inline html/changelog.html:62]: have you ever looked at the codebase? this is one big pile of "wat"

Comment (AliiTea) [inline html/changelog.html:62]: I like manually doing it more than your shitty method @Iamgoofball :^)

Comment (Iamgoofball) [inline html/changelog.html:62]: you're a really dumb nex, and you never stopped being one, now port this:
https://github.com/tgstation/-tg-station/pull/4477

**Comment by AliiTea** [comment]
@Iamgoofball no

**Comment by AliiTea** [comment]
god I hate changelogs

**Pull request closed without merging**